src/value.rs

```diff
@@ -435,6 +437,37 @@ where
     }
 }
 
+impl<'a, T> From<Cow<'a, T>> for Value
+where
+    T: Clone + AsRef<str> + Into<Value>,
+    Cow<'a, T>: From<Cow<'a, str>>,
+{
+    fn from(x: Cow<'a, T>) -> Value {
+        x.as_ref().clone().into()
+    }
+}
+
+impl<'a> ValueType for Cow<'a, str> {
```

Comment:
Why you need this implementation?

Reply:
```shell
    --> src/value.rs:1488:24
     |
1488 |         let v: Value = val.into();
     |                        ^^^ ---- required by a bound introduced by this call
     |                        |
     |                        the trait `From<std::borrow::Cow<'_, str>>` is not implemented for `value::Value`
```

Reply:
sorry, that is the other one, this one is ValueType
```shell
--> src/value.rs:1474:31
     |
1474 |         let out: Cow<str> = v.unwrap();
     |                               ^^^^^^ the trait `value::ValueType` is not implemented for `std::borrow::Cow<'_, str>`
```